cmd/otk-make-partition-mounts-devices/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/osbuild/images/internal/otk"
+	"github.com/osbuild/images/pkg/osbuild"
+)
+
+type Input struct {
+	Filename string                `json:"filename"`
+	Internal otk.PartitionInternal `json:"internal"`
```

Comment:
The comment from https://github.com/achilleas-k/images/pull/6#discussion_r1627886000 applies here as well, it sounds like we should merge this and then work in a followup on the suggestion there maybe?